WooCommerce/Resources/ar.lproj/Localizable.strings

```diff
@@ -1207,7 +1204,8 @@ which should be translated separately and considered part of this sentence. */
 /* Shown while logging in with Apple and the app waits for the site creation process to complete. */
 "Continuing with Apple" = "المتابعة باستخدام Apple";
 
-/* Conversion stat label on dashboard. */
+/* Conversion stat label on dashboard.
+   Conversion title label for the store info widget */
```

Comment:
Unfortunate that developers didn't use reverse-DNS keys when they started developing the widget (similar to [how we've done it for Wordpress widget targets](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/WordPress/Resources/en.lproj/Localizable.strings#L9492)), this will likely lead to translation clashes 😢

Reply:
Gotta make time to at least implement the SwiftLint / Peril / Danger lint to make sure new `NSLocalizedStrings` / `AppLocalizedStrings` use the reverse-DNS key format...